### PR TITLE
hide/display applyTo on weButton d

### PR DESCRIPTION
**Pull request by FrancoisGe**

### CODE REVIEW

File: addons/html_builder/static/src/builder/builder_helpers.js [hunk 1]
```diff
@@ -23,11 +23,33 @@ export class WithSubEnv extends Component {
...
+    }
+}
+
 export function useWeComponent() {
     const comp = useComponent();
     const newEnv = {};
+    const oldEnv = useEnv();
     if (comp.props.applyTo) {
-        newEnv.editingElement = comp.env.editingElement.querySelector(comp.props.applyTo);
+        newEnv.editingElement = oldEnv.getEditingElement().querySelector(comp.props.applyTo);
```
Comment (Goaman) [inline addons/html_builder/static/src/builder/builder_helpers.js]: let

File: addons/html_builder/static/src/builder/builder_helpers.js [hunk 2]
```diff
@@ -23,11 +23,33 @@ export class WithSubEnv extends Component {
...
+        useBus(oldEnv.editorBus, "BEFORE:STEP_ADDED", () => {
```
Comment (Goaman) [inline addons/html_builder/static/src/builder/builder_helpers.js]: useBus(oldEnv.editorBus, "UPDATE_EDITING_ELEMENT", () => {

File: addons/html_builder/static/src/builder/builder_helpers.js [hunk 3]
```diff
@@ -113,9 +135,12 @@ export function useClickableWeWidget() {
         return actions;
     }
     function isActive() {
+        if (!comp.env.getEditingElement()) {
```
Comment (Goaman) [inline addons/html_builder/static/src/builder/builder_helpers.js]: ?

File: addons/html_builder/static/src/builder/plugins/builder_options_plugin.js [hunk 4]
```diff
@@ -35,6 +35,10 @@ export class BuilderOptionsPlugin extends Plugin {
     }
 
     changeSidebarTarget(selectedElement) {
+        // if (this.lastSelectedElement === selectedElement) {
```
Comment (Goaman) [inline addons/html_builder/static/src/builder/plugins/builder_options_plugin.js]: 👀

File: addons/html_builder/static/tests/toolbox/widget.test.js [hunk 5]
```diff
@@ -249,6 +268,26 @@ describe("WeButtonGroup", () => {
                         <WeButton action="'customAction4'"/>
                     </WeButtonGroup>`,
         });
+        addActionOption({
```
Comment (Goaman) [inline addons/html_builder/static/tests/toolbox/widget.test.js:271]: addActionOptions

File: addons/html_builder/static/src/builder/builder_helpers.js (hunk 3 above)
Comment (FrancoisGe) [reply addons/html_builder/static/src/builder/builder_helpers.js]: we need it

**Pull request merged**